<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);

</string>
  </property>
  <widget class="QPushButton" name="close_">
   <property name="geometry">
    <rect>
     <x>1880</x>
     <y>20</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
background-color: rgb(255, 255, 255);
border-radius: 15px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icons/remove-button.png</normaloff>../icons/remove-button.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>1540</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: hidden;
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1540</width>
      <height>1000</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="settingsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>340</width>
     <height>1000</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/filter (1).png</normaloff>../icons/filter (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Filter Insights</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/filter.png</normaloff>../icons/filter.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="tuneLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Tune results</string>
    </property>
   </widget>
   <widget class="QSlider" name="temperatureSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSlider" name="termLimitSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="audacitySlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>900</y>
      <width>300</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="temperatureLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
   </widget>
   <widget class="QLabel" name="cycleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>Cycle Limit</string>
    </property>
   </widget>
   <widget class="QLabel" name="audacityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>860</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>Audacity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="subjectsCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(150, 150, 150);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>All</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Macro Economics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diplomaty</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inflation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Politics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Taxes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Employement</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Trade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>International Affairs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Market Performance</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="locationsCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(150, 150, 150);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>All</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Angola</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mozambica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>United Kingdom</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="assetsCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(150, 150, 150);</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>All</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>All</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AAPL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MSFT</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../rsrc/images/backgrounds.qrc"/>
 </resources>
 <connections/>
</ui>
